vaccine_feed_ingest_schema/location.py

```diff
@@ -4,10 +4,141 @@
+# Less strict than spec so normalizers don't need to encode phone numbers exactly
+# e.g. (444) 444-4444, +1 (444) 444-4444
+US_PHONE_RE = re.compile(
+    r"^(?:(?:\+?1\s*(?:[.-]\s*)?)?(?:\(\s*([2-9]1[02-9]|[2-9][02-8]1|[2-9][02-8][02-9])\s*\)|([2-9]1[02-9]|[2-9][02-8]1|[2-9][02-8][02-9]))\s*(?:[.-]\s*)?)?([2-9]1[02-9]|[2-9][02-9]1|[2-9][02-9]{2})\s*(?:[.-]\s*)?([0-9]{4})(?:\s*(?:\#|x\.?|ext\.?|extension)\s*(\d+))?$"
+)
+
+
+@enum.unique
+class State(str, enum.Enum):
+    ALABAMA = "AL"
+    ALASKA = "AK"
+    AMERICAN_SAMOA = "AS"
+    ARIZONA = "AZ"
+    ARKANSAS = "AR"
+    CALIFORNIA = "CA"
+    COLORADO = "CO"
+    CONNECTICUT = "CT"
+    DELAWARE = "DE"
+    DISTRICT_OF_COLUMBIA = "DC"
+    FLORIDA = "FL"
+    GEORGIA = "GA"
+    GUAM = "GU"
+    HAWAII = "HI"
+    IDAHO = "ID"
+    ILLINOIS = "IL"
+    INDIANA = "IN"
+    IOWA = "IA"
+    KANSAS = "KS"
+    KENTUCKY = "KY"
+    LOUISIANA = "LA"
+    MAINE = "ME"
+    MARYLAND = "MD"
+    MASSACHUSETTS = "MA"
+    MICHIGAN = "MI"
+    MINNESOTA = "MN"
+    MISSISSIPPI = "MS"
+    MISSOURI = "MO"
+    MONTANA = "MT"
+    NEBRASKA = "NE"
+    NEVADA = "NV"
+    NEW_HAMPSHIRE = "NH"
+    NEW_JERSEY = "NJ"
+    NEW_MEXICO = "NM"
+    NEW_YORK = "NY"
+    NORTH_CAROLINA = "NC"
+    NORTH_DAKOTA = "ND"
+    NORTHERN_MARIANA_IS = "MP"
+    OHIO = "OH"
+    OKLAHOMA = "OK"
+    OREGON = "OR"
+    PENNSYLVANIA = "PA"
+    PUERTO_RICO = "PR"
+    RHODE_ISLAND = "RI"
+    SOUTH_CAROLINA = "SC"
+    SOUTH_DAKOTA = "SD"
+    TENNESSEE = "TN"
+    TEXAS = "TX"
+    UTAH = "UT"
+    VERMONT = "VT"
+    VIRGINIA = "VA"
+    VIRGIN_ISLANDS = "VI"
+    WASHINGTON = "WA"
+    WEST_VIRGINIA = "WV"
+    WISCONSIN = "WI"
+    WYOMING = "WY"
+
+
+@enum.unique
+class ContactType(str, enum.Enum):
+    GENERAL = "general"
+    BOOKING = "booking"
+
+
+@enum.unique
+class DayOfWeek(str, enum.Enum):
+    MONDAY = "monday"
+    TUESDAY = "tuesday"
+    WEDNESDAY = "wednesday"
+    THURSDAY = "thursday"
+    FRIDAY = "friday"
+    SATURDAY = "saturday"
+    SUNDAY = "sunday"
+    PUBLIC_HOLIDAYS = "public_holidays"
+
+
+@enum.unique
+class VaccineType(str, enum.Enum):
+    PFIZER_BIONTECH = "pfizer_biontech"
+    MODERNA = "moderna"
+    JOHNSON_JOHNSON_JANSSEN = "johnson_johnson_janssen"
+    OXFORD_ASTRAZENECA = "oxford_astrazeneca"
+
+
+@enum.unique
+class WheelchairAccessLevel(str, enum.Enum):
+    YES = "yes"  # there is wheelchair access not sure about level of service
+    FULL = "full"  # here is full wheelchair access
+    PARTIAL = "partial"  # there is partial wheelchair access
+    NO = "no"  # there is no wheelchair access
+
+
+@enum.unique
+class VaccineProvider(str, enum.Enum):
+    """Parent organization that provides vaccines"""
+
+    RITE_AID = "rite_aid"
+    WALGREENS = "walgreens"
+    SAFEWAY = "safeway"
+    VONS = "vons"
+    SAMS = "sams"
+    ALBERTSONS = "albertson"
+    PAVILIONS = "pavilions"
+    WALMART = "walmart"
+    CVS = "CVS"
```

Comment:
should this be all-lower (like the others)?

Reply:
yes. good catch